src/lib/tenant/configs/contracts/ens.ts

```diff
@@ -5,11 +5,19 @@ import {
 import { ITokenContract } from "@/lib/contracts/common/interfaces/ITokenContract";
 import { TenantContract } from "@/lib/tenant/tenantContract";
 import { TenantContracts } from "@/lib/types";
-import { ethProvider, sepoliaProvider } from "@/app/lib/provider";
 import { mainnet, sepolia } from "viem/chains";
 import { IGovernorContract } from "@/lib/contracts/common/interfaces/IGovernorContract";
+import { AlchemyProvider } from "ethers";
 
-export const ensTenantContractConfig = (isProd: boolean): TenantContracts => {
+interface Props {
+  isProd: boolean;
+  alchemyId: string;
+}
+
+export const ensTenantContractConfig = ({
+  isProd,
+  alchemyId,
```

Comment:
do we think our alchemyId will change between tenants or can we just do something like `process.env.ALCHEMY_ID` rather than passing it through to each tenant's config?

Reply:
I don't think so. I also don't have a strong preference here but my initial approach was to minimize the number of places in which `process.env.` vars are referenced.